localstack/services/dynamodb/provider.py

```diff
@@ -1197,6 +1205,56 @@ def describe_kinesis_streaming_destination(
             KinesisDataStreamDestinations=stream_destinations, TableName=table_name
         )
 
+    #
+    # Continuous Backups
+    #
+
+    def describe_continuous_backups(
+        self, context: RequestContext, table_name: TableName
+    ) -> DescribeContinuousBackupsOutput:
+        self.get_global_table_region(context, table_name)
```

Comment:
Should we take the result of this call and use it as the region for the store? (same pattern we're also using for other methods in the provider..)
```
        global_table_region = self.get_global_table_region(context, table_name)
        store = get_store(context.account_id, region=global_table_region)
```

Reply:
I am actually not sure about the behavior here. 
If I remember correctly our implementation of global tables, we actually keep in DynamoDB local a single table in a given region. When we receive a request from a global region, we check with `get_global_table_region` which is the region that actually holds the data in DDBLocal, and forwards the calls to that specific region with `self._forward_request(context=context, region=global_table_region)` (we basically modify the region in the context to hit the "original" region). All over the provider, we use this `global_table_region` only to patch the forwarded request, but we access the store with the actual region in the context.